Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexProperty.java

```diff
@@ -408,6 +408,7 @@ public void assignProperties(Map<String, String> source) {
                     this.assignTimeDuration(source);
                     break;
                 default:
+                    continue;
```

Comment:
[C#](https://github.com/microsoft/Recognizers-Text/pull/2457/files#diff-30c57aa18922d5b3781bb097653c619215ad8ab665f8dec20d89ce8cabafec1fR203) checks if the value is empty so the `switch` is not executed. Here we are not doing this.